include/ppx/knob.h

```diff
@@ -314,6 +316,67 @@ class KnobDropdown final
     std::vector<T> mChoices;
 };
 
+// KnobConstant is intended for parameters that cannot be adjusted when the application is run
+// They will not be displayed in the UI and cannot have their values changed by the application
+template <typename T>
+class KnobConstant final
+    : public Knob
+{
+public:
+    KnobConstant(const std::string& flagName, T defaultValue)
+        : Knob(flagName, false)
+    {
+        SetValue(defaultValue);
+    }
+
+    KnobConstant(const std::string& flagName, T defaultValue, T minValue, T maxValue)
```

Comment:
I feel like having the validator function is a good idea (since those would replace flags AFAIK).
But seems like the minValue/maxValue constraints this constant to be numerical, vs being some string value by ex.
Do you think it would be useful to have the generic KnobConstant() take a unary function for validator, and maybe have a template specialization for numerical values with the minValue/maxValue constructor you have here?